bitnami/contour/values.yaml

```
@@ -0,0 +1,234 @@
+  #   - "response_flags"
+  #   - "uber_trace_id"
+  #   - "upstream_cluster"
+  #   - "upstream_host"
+  #   - "upstream_local_address"
+  #   - "upstream_service_time"
+  #   - "user_agent"
+  #   - "x_forwarded_for"
+
+## String to partially override contour.fullname include (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override contour.fullname template
+##
+# fullnameOverride:
+
+## Number of contour Pod replicas
+##
+replicaCount: 2
+
+rbac:
+  # create specifies whether to install and use RBAC rules.
+  create: true
+
+contour:
+  image:
+    registry: docker.io
+    repository: projectcontour/contour
+    tag: v1.2.1
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+
+  ## Contour container resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## ref: https://projectcontour.io/guides/resource-limits/
+  ##
+  resources:
+    ## We usually recommend not to specify default resources and to leave this as a conscious
+    ## choice for the user. This also increases chances charts run on environments with little
+    ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+    ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits: {}
+      # cpu: 400m
+      # memory: 250Mi
+    requests: {}
+      # cpu: 100m
+      # memory: 25Mi
+
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  serviceAccount:
+    # Specifies whether a ServiceAccount should be created
+    create: true
+    # The name of the ServiceAccount to use.  If not set and create is
+    # true, a name is generated using the fullname template
+    name: ""
+
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 15
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 3
+    successThreshold: 1
+
+  securityContext:
+    enabled: true
+    runAsNonRoot: true
+    runAsUser: 65534
+    runAsGroup: 65534
+
+  certgen:
+    serviceAccount:
+      # Specifies whether a ServiceAccount should be created
+      create: true
+      # The name of the ServiceAccount to use.  If not set and create is
+      # true, a name is generated using the fullname template
+      name: ""
+
+envoy:
+  image:
+    registry: docker.io
+    repository: envoyproxy/envoy
+    tag: v1.13.1
```

Once finished the image on our side, it should be looks like:
```suggestion
    registry: docker.io
    repository: bitnami/envoy
    tag: 1.13.1-debian-10-r0
```

BTW, we planned for the next weeks the contour and metallb image, this one was not on our radar. So I will add it to our TODO list